src/maps/maps.controller.ts

```diff
@@ -20,33 +20,34 @@ export class MapsController {
     return formattedMaps
   }
 
+  // TODO What are we going to return here?
```

Comment:
All known map keys? IDK. I created it before taking a look at what IMap provides :)